javaAdvanced_iteratorsAndComparators/src/lab/book/Main.java

```diff
@@ -0,0 +1,32 @@
+package lab.book;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Create a class Book from UML diagram below: Book - title: String - year: int
+ * - authors: List<String> - setTitle(String) - setYear(String) -
+ * setAuthors(String�) + getTitle(): String + getYear(): int + getAuthors():
+ * List<String> You can use only one constructor. Authors can be anonymous, one
+ * or many.
+ * 
+ * @author Ivelin Gavrailov
+ * @since 10.11.2020
+ *
+ */
+public class Main {
+
+	public static void main(String[] args) {
+		Book bookOne = new Book("Animal Farm", 2003, "George Orwell");
+		Book bookThree = new Book("The Documents in the Case", 2002);
+		Book bookTwo = new Book("The Documents in the Case", 1930, "Dorothy Sayers", "Robert Eustace");
+
+		List<Book> books = new ArrayList<>();
+		books.add(bookOne);
+		books.add(bookTwo);
+		books.add(bookThree);
+		for (Book book : books) {
```

Comment:
This print is not working. You didn't implement toString() in Book.

Reply:
Auto generated toString() in Book and System.out.println(books); will works fine

Reply:
fixed